content/en/docs/use-cases/openshift.md

```diff
@@ -44,18 +44,44 @@ oc login -u kubeadmin -p <your password> https://api.crc.testing:6443
 
 ### Security Context Constraints
 
-Before installing Flux, you need to set the **anyuid** SCC for Kustomization and Helm controllers,
-and the **privileged** SCC for the other 4 controllers in the `flux-system` namespace, like this:
+Before installing Flux with CLI, you need to set the **nonroot** SCC for all controllers in the `flux-system` namespace, like this:
 
 ```sh
 NS="flux-system"
-oc adm policy add-scc-to-user anyuid system:serviceaccount:$NS:kustomize-controller
-oc adm policy add-scc-to-user anyuid system:serviceaccount:$NS:helm-controller
+oc adm policy add-scc-to-user nonroot system:serviceaccount:$NS:kustomize-controller
+oc adm policy add-scc-to-user nonroot system:serviceaccount:$NS:helm-controller
+oc adm policy add-scc-to-user nonroot system:serviceaccount:$NS:source-controller
+oc adm policy add-scc-to-user nonroot system:serviceaccount:$NS:notification-controller
+oc adm policy add-scc-to-user nonroot system:serviceaccount:$NS:image-automation-controller
+oc adm policy add-scc-to-user nonroot system:serviceaccount:$NS:image-reflector-controller
+```
 
-oc adm policy add-scc-to-user privileged system:serviceaccount:$NS:source-controller
-oc adm policy add-scc-to-user privileged system:serviceaccount:$NS:notification-controller
-oc adm policy add-scc-to-user privileged system:serviceaccount:$NS:image-automation-controller
-oc adm policy add-scc-to-user privileged system:serviceaccount:$NS:image-reflector-controller
+Also, you have to patch your Kustomization to remove the SecComp Profile and enforce runUserAs to the same UID provided by the images to prevent OpenShift to alter the user expected by our controllers.
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - gotk-components.yaml
+  - gotk-sync.yaml
+patches:
+  - patch: |
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: all
+      spec:
+        template:
```

Comment:
Don't we need to remove the `fsGroup` like showed here https://fluxcd.io/docs/cheatsheets/bootstrap/#openshift-compatibility

Reply:
No we don't need to. The removal in the cheat sheet is an over doing.

Reply:
Here's a confirmation:
![image](https://user-images.githubusercontent.com/10666/154966969-b515d0ee-14da-452c-83fc-26c1d0e0f9bb.png)

Reply:
Why did we have to remove it before? Does `nonroot` allows fsGroup?

Reply:
We needed to remove it from Flux Operator on OpenShift because I didn't know a way to add `nonroot` or any SCC there in the CSV.
After digging more, I found a way to allow SCC to each controller by having rule like this:

https://github.com/weaveworks/flux2-openshift/blob/main/templates/clusterserviceversion.yaml#L502

Reply:
Are yo saying that we no longer need to remove `fsGroup` in `flux2-openshift` repo?